Framework/PythonInterface/test/python/plugins/algorithms/VesuvioCorrectionsTest.py

```diff
@@ -27,6 +31,15 @@ def setUp(self):
         if self._test_container_ws is None:
             self.__class__._test_container_ws = create_test_container_ws()
 
+        self._input_bins = self._test_ws.blocksize()
+
+        # Check OS for RHEL7 or Ubuntu
+        dist = platform.linux_distribution()
```

Comment:
I just checked the return values of `platform.linux_distribution()` on Windows and you get a tuple with 3 entries but each entry is an empty string. I think a better test here would be `self._is_linux = any(platform.linux_distribution())` as it doesn't limit it to RHEL7 or Ubuntu.